packages/web/src/components/collection/CollectionCard.tsx

```diff
@@ -51,50 +59,100 @@ export const CollectionCard = (props: CollectionCardProps) => {
     permalink,
     playlist_owner_id,
     repost_count,
-    save_count
+    save_count,
+    is_private,
+    access,
+    stream_conditions
   } = collection
 
+  const isPurchase = isContentUSDCPurchaseGated(stream_conditions)
+
+  const dogEarType = is_private
+    ? DogEarType.HIDDEN
+    : isPurchase && !access.stream
+    ? DogEarType.USDC_PURCHASE
+    : null
+
   return (
     <Paper
       role='button'
       tabIndex={0}
       onClick={handleClick}
-      aria-labelledby={`${id}-title ${id}-artist`}
+      aria-labelledby={`${id}-title ${id}-artist ${id}-repost ${id}-favorite ${id}-condition`}
       direction='column'
+      border='default'
       w={cardSizes[size]}
-      css={{ cursor: 'pointer' }}
+      css={{ cursor: 'pointer', overflow: 'unset' }}
     >
+      {dogEarType ? (
+        <DogEar type={dogEarType} css={{ top: -1, left: -1 }} />
+      ) : null}
       <Flex direction='column' p='s' gap='s'>
         <CollectionImage
           collectionId={id}
           size={cardSizeToCoverArtSizeMap[size]}
           data-testid={`${id}-cover-art`}
         />
-        <Link id={`${id}-title`} to={permalink} css={{ alignSelf: 'center' }}>
-          <Text variant='title' tag='span' color='default' textAlign='center'>
+        <Text
+          id={`${id}-title`}
+          variant='title'
+          color='default'
+          ellipses
+          asChild
+        >
+          <Link to={permalink} css={{ pointerEvents: 'none' }}>
             {playlist_name}
-          </Text>
-        </Link>
+          </Link>
+        </Text>
```

Comment:
Is there not a `CollectionLink` component for this? Or is this one different from the standard links?

Reply:
good question! we could use collection-link and pass the pointer-events none to it. that's probably the move honestly. collection link doesn't exist currently though, but ill add one as i start adding collection cards in